amy/templates/emails/email_template_edit.html

```diff
@@ -1,9 +1,23 @@
 {% extends "base_nav.html" %}
 
 {% load crispy_forms_tags %}
+{% load emails %}
 
 {% block content %}
 
 {% crispy form %}
 
+<div class="row">
+  <div class="col-12 col-lg-2">
+    <strong>Template variables</strong>
+  </div>
+  <div class="col-12 col-lg-10">
```

Comment:
I'm going to update the wording so that it's clearer what they refer to.

About template validation: yes, I think I could build that in here. The previous system contained similar validation (for proper Django template format).

Reply:
@elichad Actually the validation is not possible at this point.

We'd get the best results if we used Django Template Engine behavior for checking if all variables are accessible. However, to do that we have to have context with real data. At this point we're working with email templates without any context, so proper validation is not possible.